bigtable-hbase-2.x-parent/bigtable-hbase-2.x-shaded/pom.xml

```diff
@@ -109,6 +109,11 @@ limitations under the License.
       <artifactId>metrics-core</artifactId>
       <version>${dropwizard.metrics.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-metrics-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
please move this up to just before jsr305 (it should be the first in the manually promoted list)
but hbase-shaded-client is out of place. This is fixed in master